src/external-types/reveal.js/index.d.ts

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
I guess `@types/reveal` was not sufficient?

Reply:
They're exporting it wrong.